### DISCUSSION TITLE
[website] rename "Integrations"?

### DISCUSSION THREAD

**Discussion by dmaskasky** [pmndrs/jotai#2169] [General]
Hello, I have a new library [jotai-effect](https://github.com/jotaijs/jotai-effect), I'd like to add its docs to the website. The most appropriate place is under integrations, but I feel integrations is a bit of a misnomer at this point. To me, it feels more like 3rd party repos, than library integrations. It would be nice to pick a section label that would be more accurate to describe these packages.

One suggestion I have is **Ecosystem**. I feel this describes the type of links we'd like to put in here. What do you think? 

@sandren @dai-shi

**Comment by dai-shi** [comment]
I like it.

**Comment by sandren** [comment]
> Hello, I have a new library [jotai-effect](https://github.com/jotaijs/jotai-effect), I'd like to add its docs to the website. The most appropriate place is under integrations, but I feel integrations is a bit of a misnomer at this point. To me, it feels more like 3rd party repos, than library integrations. It would be nice to pick a section label that would be more accurate to describe these packages.
> 
> One suggestion I have is **Ecosystem**. I feel this describes the type of links we'd like to put in here. What do you think?
> 
> @sandren @dai-shi

That's a good point. Otherwise people might expect that your library is an integration with the `effect` package. I'll think a bit more about the label. 🤔

**Comment by dmaskasky** [comment]
For your consideration: https://github.com/pmndrs/jotai/pull/2170
